task/control-plane-agent/Cargo.toml

```diff
@@ -14,6 +14,9 @@ ssmarshal.workspace = true
 static_assertions.workspace = true
 zerocopy.workspace = true
 
+p256 = { workspace = true, optional = true }
+sha2 = { workspace = true, optional = true }
```

Comment:
We have a mix of SHA2 and SHA3 in our implementations with some preference for SHA3.
If this is an arbitrary choice, then preference would be for SHA3.

Reply:
The current support keys are using `ecdsa-sha2-nistp256` certificates, so I think we're stuck with SHA2 for now.

Reply:
FWIW I think this isn't just the support keys; the ssh-agent (our mechanism for talking to support yubikeys) protocol only lists SHA2-256 and SHA2-512 as options (https://datatracker.ietf.org/doc/html/draft-miller-ssh-agent-04#section-7.3).